### ISSUE TITLE
Contextual results are missing their context description

### ISSUE THREAD

**Issue by bentookey-yotta** [easyops-cn/docusaurus-search-local#424]
We're currently using Docusaurus v3.2 and easyops v0.40.1 with contextual search configured.

In v0.44, when searching within a context (e.g. _the "Concepts" section_), most search results are lacking their context description and have incomplete breadcrumbs. The standard "Everywhere" context seems unaffected.

This was introduced in **v0.41** presumably by https://github.com/easyops-cn/docusaurus-search-local/issues/344.

### v0.44:
![easyops-v44](https://github.com/easyops-cn/docusaurus-search-local/assets/105360709/31ff02a8-b523-45af-a57b-9d24ae9797b3)

### v0.40
![live](https://github.com/easyops-cn/docusaurus-search-local/assets/105360709/b321043d-a8e9-4ac6-b779-90a7f0b015bd)

### Config
```
      {
        docsRouteBasePath: '/',
        indexDocs: true,
        indexBlog: false,
        indexPages: false,
        explicitSearchResultPath: true,
        searchBarShortcutHint: true,
        // `hashed` is recommended as long-term-cache of index file is possible.
        hashed: true,
        searchContextByPaths: [
          { label: 'Concepts', path: 'basics/fundamentals' },
          { label: 'Alloy Web', path: 'web' },
          { label: 'Alloy Mobile', path: 'mobile' },
        ],
        useAllContextsWithNoSearchContext: true,
        ignoreCssSelectors: '.search-ignore',
        ignoreFiles: [/.*getting-started.*\/alloy-search.*/],
      },
```

Unfortunately, this blocks us from updating Docusaurus (as v3.4 introduces the "duplicate URL" bug solved in v0.41.1). Hopefully, it can be resolved in due course 🙂

**Comment by weareoutman** [comment]
Did you specified `keywords` in front matter of your relevant docs?

Currently, keywords take precedence over content in search index, which would cause search results lacking content part. But this should not be affected by searching within or without a context.

Could you share a repro link?

**Comment by bentookey-yotta** [comment]
Thanks for the quick response. Our repo is private but I've uploaded an affected build to Netlify (v0.44, Docusaurus v3.4): [magical-sopapillas-59e9f2.netlify.app](https://magical-sopapillas-59e9f2.netlify.app/)
Live site: [help.alloyapp.io](https://help.alloyapp.io)

None of our docs include the `keywords` or `tags` front matter.

**Comment by weareoutman** [comment]
Fixed in 0.44.1

Thanks for reporting.
